src/sui-card-primary/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
These "SuiCardPrimaryContent" and "SuiCardSecondaryContent" I think are superfluous (and both do the same, wraps into a div the primary/secondary props). What about if we get rid of them and in the "sui-card/index.jsx" just simply this wrapper div with the content? It will be more clean and clear imho :)